### ISSUE TITLE
XYFrame Crashes when sent an empty array

### ISSUE THREAD

**Issue by ZachHaber** [emeeks/semiotic#178]
![semiotic 1 9 xyframe breaking](https://user-images.githubusercontent.com/19843782/37493195-21b837e2-2872-11e8-95eb-0037ddcdebbd.PNG)
This started when I updated from 1.8.3, I've isolated it to starting in 1.9.0.
I believe the failure occurs in the new accessibility features when coordinates is set to an empty array. (i.e.) an empty graph. The accessibility features tries to access undefined values and crashes.

**Comment by emeeks** [comment]
Fixed in the next version
